test/1.4/misc/T_dmllib.dml

```diff
@@ -0,0 +1,450 @@
+/*
+  © 2022 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+dml 1.4;
+device test;
+import "testing.dml";
+
+header %{
+    bool dmllib_test(char *);
+%}
+
+extern bool dmllib_test(char *);
+
+method test() throws {
+    local char msg_buf[512];
+    if (!dmllib_test(msg_buf)) {
+        log error: "%s", msg_buf;
+    }
+}
+
+footer %{
```

Comment:
nicer for editors if you extract this to a separate .c or .h file that you `#include`

Reply:
Done.